public/app/views/pdf/_archival_object.html.erb

```diff
@@ -54,38 +54,20 @@ end
             <% if instance['sub_container'] %>
                 <div class="container-information">
                     <%
-                      if instance['sub_container'] &&
-                         instance['sub_container']['top_container'] &&
-                         instance['sub_container']['top_container']['_resolved'] &&
-                         instance['sub_container']['top_container']['_resolved']['display_string']
-
-                        # Extract just the container type and indicator from the display string
-                        # The display string format is: "Type Indicator: series_info [barcode_info]"
-                        # We want to extract "Type Indicator" part only
+                      box_string = lambda {
+                        return '' unless instance.dig('sub_container', 'top_container', '_resolved', 'display_string')
+
                         tc_resolved = instance['sub_container']['top_container']['_resolved']
-                        if tc_resolved['type'] && tc_resolved['indicator']
-                          box_string = "#{tc_resolved['type'].capitalize} #{tc_resolved['indicator']}"
-                        else
-                          # Fallback to parsing display_string
-                          tc_display_string = tc_resolved['display_string'].split(':')[0]
-                          if tc_display_string =~ /:/
-                            box_string = tc_display_string.split(':')[0]
```

Comment:
@cpapazoglou I don't think `if tc_display_string =~ /:/` ever returns `true` since we just split on the first semicolon above, so `tc_display_string` is always the value of `box_string` in this `else` fallback.